salesagility/SuiteCRM
# Modifying calendar to support additionalDetails

## Description







This change resolves issue #2347. Developers wish for the additional details to added back into qtip popups. I have also added a change which makes use of the smarty tpls, and a change which handles when you click on an event in the calendar.
## Motivation and Context


- To keep developers and users happy.
- Give developers more control over what is displayed inside the qtips. 
## How To Test This


- Run Repair and rebuild
- Go to calendar
- You when hovering over an activity is should show a loading text and then the content of the activity. This may happen so fast that you will miss it. So If you see the qtip change in size that is also an indication.
## Types of changes


### Final checklist
